### DISCUSSION TITLE
Inconsistent timer

### DISCUSSION THREAD

**Discussion by melonedo** [fonsp/Pluto.jl#1035] [Questions about using Pluto] [answered]
When I am creating a plot, the cell timer did not stop until about 30s, but after it stopped, the displayed time is 19.3s, is this intentional?

**Comment by fonsp** [comment, marked as answer]
It is! We chose to only display the time that it took to run a cell, excluding the time that it took to display the result. In the case of Plots.jl, a lot of code is being compiled when you _show_ the plot object.
